concrete-core/src/specification/engines/ggsw_ciphertext_trivial_encryption.rs

```diff
@@ -0,0 +1,51 @@
+use crate::specification::engines::{engine_error, AbstractEngine};
+use crate::specification::entities::{GgswCiphertextEntity, PlaintextEntity};
+use concrete_commons::parameters::{
+    DecompositionBaseLog, DecompositionLevelCount, GlweSize, PolynomialSize,
+};
+
+engine_error! {
+    GgswCiphertextTrivialEncryptionError for GgswCiphertextTrivialEncryptionEngine @
+}
+
+/// A trait for engines trivially encrypting GGSW ciphertext.
+///
+/// # Semantics
+///
+/// This [pure](super#operation-semantics) operation generates a GGSW ciphertext containing the
+/// trivial encryption of the `input` plaintext vector with the requested `glwe_size`.
```

Comment:
Plaintext vector?

Reply:
Check also below.

Reply:
It should be "plaintext", bad cop-pasting.